tests/physics/code_runaway/code_runaway.py

```diff
@@ -130,7 +126,7 @@ def run(args):
     nE, nT = T.shape
     rr     = np.zeros((nE, nT))
     rrFull = np.zeros((nE, nT, nt))
-    for i in range(0, nE):
+    for i in range(1, nE):
```

Comment:
I am looking over this again. I struggle with reproducing the runaway test in the new branch, and I _suspect_ that in master we only get agreement because the DREAM simulation is underresolved by just the right amount. I'll try to verify this. 

We obviously shouldn't skip the first points like I do here, but perhaps instead look at increasing the tolerance of the test (in case I am not mistaken here).

Reply:
^---- nope, this was not it. I'll keep looking for an explanation..

Reply:
I think I've figured it out. On the new branch, the bulk is being depleted so that n_cold decreases in the cases where n_re is the largest. Thus, the friction decreases (and hence the effective Dreicer field, as we saw in NORSE) and the runaway rate increases  In master, ncold is kept constant, and this does not happen. I will try to solve it by prescribing ncold instead of evaluating it self-consistently.

Reply:
fixed this in the latest commit